### PR TITLE
frontend: fix build time sorting

### LINKED ISSUE #2519: Web UI, sorting builds by build time does not work
screenshot

The sort appears to be quite arbitrary. When I sort in the other order, I get the pending builds first and the order seems to reverse-match the one on the screenshot (but I haven't checked that thoroughly).

### LINKED ISSUE #2529: The "automatic tagging" to project action doesn't work
ssia

### PR DESCRIPTION
**Pull request by nikromen**
Fixes #2519

### CODE REVIEW

**Comment by praiskup** [comment]
LGTM

**Comment by praiskup** [comment]
Even though I'd appreciate better explanation how changing str to int in the output helps.  It shouldn't matter, because jinja anyways expands the value to string.  What am I missing?

**Comment by nikromen** [comment]
if it expands to string again then the problem remains... I haven't tested it yet... it is just a blind solution... sorry for not converting it to draft

the reason why it is wrong is because it is comparing string, not numbers

Screenshot_20230208_092014

**Comment by nikromen** [comment]
tested it, it's working... for explanation see commit message

File: frontend/coprs_frontend/coprs/filters.py [hunk 1]
```diff
@@ -137,18 +137,18 @@ def localized_time(time_in, timezone):
...
 @app.template_filter('timestamp_diff')
 def timestamp_diff(time_in, until=None):
-    """ returns string with difference between two timestamps
+    """ returns difference between two timestamps
 
     Input is in EPOCH (seconds since epoch).
     """
     if time_in is None:
-        return " - "
+        return 0
```
[the remark this answers is not in the document]
Comment (praiskup) [reply frontend/coprs_frontend/coprs/filters.py:145]: Ah, it's the `started_on`, which would be `None` indeed.

**Review by praiskup — APPROVED** [review]

**Comment by hroncok** [comment]
Thanks

**Pull request merged**